libraries/AP_ICEngine/AP_ICEngine.cpp

```diff
@@ -422,9 +436,9 @@ void AP_ICEngine::update(void)
             float rpm_value;
             if (!AP::rpm()->get_rpm(rpm_instance-1, rpm_value) ||
                 rpm_value < rpm_threshold) {
-                // engine has stopped when it should be running
-                state = ICE_START_DELAY;
-                GCS_SEND_TEXT(MAV_SEVERITY_INFO, "Uncommanded engine stop");
+                    // engine has stopped when it should be running
+                    state = ICE_START_DELAY;
+                    GCS_SEND_TEXT(MAV_SEVERITY_INFO, "Uncommanded engine stop");
             }
```

Comment:
Whitespace change outside of scope of this PR, but also fixed incorrectly. For our astyle rules, it should actually be:

```
            if (!AP::rpm()->get_rpm(rpm_instance-1, rpm_value) ||
                    rpm_value < rpm_threshold) {
                // engine has stopped when it should be running
                state = ICE_START_DELAY;
                GCS_SEND_TEXT(MAV_SEVERITY_INFO, "Uncommanded engine stop");
            }
  ```
  
  i.e., adding an extra indent to line 438, not 439-441

Reply:
gitkraken view messed this up for me.....  thanks for pointing it,,,